app/build.gradle.kts

```diff
@@ -1,6 +1,12 @@
 plugins {
     id("com.android.application")
     id("org.jetbrains.kotlin.android")
+    id("kotlin-kapt")
```

Comment:
do we need here kapt we use ksp in project?

Reply:
fixed